ethstorage/log/cli.go

```diff
@@ -53,22 +54,40 @@ func (cfg CLIConfig) Check() error {
 	}
 
 	level := strings.ToLower(cfg.Level)
-	_, err := log.LvlFromString(level)
+	_, err := LevelFromString(level)
 	if err != nil {
 		return fmt.Errorf("unrecognized log level: %w", err)
 	}
 	return nil
 }
 
+func LevelFromString(lvlString string) (slog.Level, error) {
+	lvlString = strings.ToLower(lvlString) // ignore case
+	switch lvlString {
+	case "trace", "trce":
+		return log.LevelTrace, nil
+	case "debug", "dbug":
+		return log.LevelDebug, nil
+	case "info":
+		return log.LevelInfo, nil
+	case "warn":
+		return log.LevelWarn, nil
+	case "error", "eror":
+		return log.LevelError, nil
+	case "crit":
+		return log.LevelCrit, nil
+	default:
+		return log.LevelDebug, fmt.Errorf("unknown level: %v", lvlString)
+	}
+}
+
 func NewLogger(cfg CLIConfig) log.Logger {
-	handler := log.StreamHandler(os.Stdout, Format(cfg.Format, cfg.Color))
-	handler = log.SyncHandler(handler)
-	handler = log.LvlFilterHandler(Level(cfg.Level), handler)
-	// Set the root handle to what we have configured. Some components like go-ethereum's RPC
-	// server use log.Root() instead of being able to pass in a log.
-	log.Root().SetHandler(handler)
-	logger := log.New()
-	logger.SetHandler(handler)
+	l, err := LevelFromString(cfg.Level)
+	if err != nil {
+		panic(err)
+	}
+	h := log.NewTerminalHandlerWithLevel(os.Stdout, l, cfg.Color)
```

Comment:
How can we support other format like "logfmt", "json", etc.

Reply:
Filed https://github.com/ethstorage/es-node/issues/374